barman/output.py

```diff
@@ -676,11 +676,20 @@ def result_list_backup(self, backup_info, backup_size, wal_size, retention_statu
             self.info(backup_info.backup_id)
             return
 
-        out_list = ["%s %s " % (backup_info.server_name, backup_info.backup_id)]
+        # Backup type can be either full, incremental or rsync
+        if backup_info.backup_manager.config.backup_method == "rsync":
```

Comment:
We cannot use the config object from the manager because it contains the current value of `backup_method` for the server.

For example, if you do the following you will have a buggy output:

* Configure server with method `rsync`
* Take a backup
* Change configuration to use method `postgres`
* Execute `barman list-backup all`

The output will show the label `F`, when in fact the backup was taken with `rsync`.

In other words, do not rely on configuration to set the label, but on the info stored in the `backup.info` at backup time.